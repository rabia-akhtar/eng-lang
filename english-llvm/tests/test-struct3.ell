struct Doc {
	string File;
	int Word_count;
};

int main()
{
  struct Doc doc;
  struct Doc essay;
  doc.File = "this is an essay";
  doc.Word_count = 20;
  essay = doc;
  print_string(essay.File);
  print(essay.Word_count);
  return 0;
}