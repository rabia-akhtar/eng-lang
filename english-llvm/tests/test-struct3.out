this is an essay
20